### ISSUE TITLE
command+enter shortcut shows confirm

### ISSUE THREAD

**Issue by paulmolluzzo** [paulmolluzzo/running-redmine#3]
When editing an issue and using the <kbd>command</kbd>+<kbd>enter</kbd> shortcut there is a confirmation prompt that you'll lose the edits. Confirming (Clicking OK) navigates away and doesn't save the edit.

**Comment by travisdoherty** [comment]
- Opened an issue.
- Hit `e` to edit.
- Added a comment. Hit `Cmd+Enter`
- Have this popup box:

![screen shot 2016-06-06 at 10 35 08](https://cloud.githubusercontent.com/assets/6044298/15825619/868dd850-2bd2-11e6-82d6-6ff898c645d0.png)
- Press `Esc` and was back at the edit form. Pressing `Cmd+Enter` no longer brings up the box.
- Reloaded page, repeat above steps, and:
- Press `Enter` to accept, and my change was saved and the page reloaded.

**Comment by paulmolluzzo** [comment]
So if I understand correctly accepting this navigation actually saved the update?

> Press Esc and was back at the edit form. Pressing Cmd+Enter no longer brings up the box.

This sounds like the keyboard shortcut lost its binding to the element. Does that seem to fit with the bug you found?

**Comment by paulmolluzzo** [comment]
> So if I understand correctly accepting this navigation actually saved the update?

Yes, confirming the navigation _does_ actually save the update, but it's kind of scary to attempt. This is also a fickle thing and I only see it occasionally. I think it's when typing the comment is the last modification I make.
